app/models/post_action_type.rb

```diff
@@ -1,6 +1,23 @@
 # frozen_string_literal: true
 
 class PostActionType < ActiveRecord::Base
+  POST_ACTION_TYPE_ALL_FLAGS_KEY = "post_action_type_all_flags"
+  POST_ACTION_TYPE_PUBLIC_TYPE_IDS_KEY = "post_action_public_type_ids"
+  LIKE_POST_ACTION_ID = 2
+  ATTRIBUTE_NAMES = %i[
+    id
+    name
+    name_key
+    description
+    notify_type
+    auto_action_type
+    require_message
+    applies_to
+    position
+    enabled
+    score_type
+  ]
+
   after_save :expire_cache
   after_destroy :expire_cache
```

Comment:
Just noticed this, why do we only expire the fragment cache and not the Redis cache?